src/exec.cpp

```diff
@@ -1031,43 +1031,6 @@ void exec_job(parser_t &parser, job_t *j) {
         }
     }
 
-    // When running under WSL, create a keepalive process unconditionally if our first
```

Comment:
There is absolutely no way somebody is running "old WSL"?

Reply:
Of course there is. But are we going to burden our codebase forever with junk working around bugs while WSL reaches maturity? It's a tradeoff between when fish 3.0 gets and released and adoption of the latest WSL releases as time goes on, and it's easy enough to tell users running into pgrp-related errors (`tcsetpgrp`/`setpgid`, primarily) under WSL to update their Windows.

Keep in mind that WSL isn't on by default under Windows 10 and it's only the most tech savvy Windows users that will know what WSL is and use it.

Reply:
I actually did not know that! I thought it was an actual thing now, with releases and such!